examples/connext_dds/logging_config/README.md

```diff
@@ -0,0 +1,122 @@
+    #09    ./logging_publisher() [0x403039]
+[2020-11-27 13:51:52.504785] U00007f753124c700_logging_publish [0x01016FA9,0xB965C9C1,0x4E59B461:0x80000003{E=DW,T=Example logging,C=logging,D=0}|WRITE]
+  Mx0D:/home/pres.1.0/srcC/writerHistoryDriver/WriterHistoryDriver.c:782:PRESWriterHistoryDriver_addWrite:RTI0x20d9006:!timestamp order in topic 'Example logging'
+[2020-11-27 13:51:52.504805] U00007f753124c700_logging_publish [0x01016FA9,0xB965C9C1,0x4E59B461:0x80000003{E=DW,T=Example logging,C=logging,D=0}|WRITE]
+  Mx0D:/home/pres.1.0/srcC/psService/PsReaderWriter.c:3850:PRESPsWriter_writeInternal:RTI0x2000008:!timestamp order
+```
+
+## Goal of the Example
+
+The goal of this example is to understand the different fields of
+**NDDS_Config_LogPrintFormat**. Reviwing the firt warning:
+
+-   **Backtrace**: It is a list of the function calls that are currently active in
+  a thread.
+
+```plaintext
+    Backtrace:
+    #4    /home/rti_connext_dds-6.1.0/lib/x64Linux3gcc5.4.0/libnddscored.so(WriterHistoryMemoryPlugin_addSample+0x68f) [0x7f752ff1f00d]
+    #5    /home/rti_connext_dds-6.1.0/lib/x64Linux3gcc5.4.0/libnddscored.so(PRESWriterHistoryDriver_addWrite+0xca7) [0x7f7530239e34]
+    #6    /home/rti_connext_dds-6.1.0/lib/x64Linux3gcc5.4.0/libnddscored.so(PRESPsWriter_writeInternal+0x2222) [0x7f75300e112e]
+    #7    /home/rti_connext_dds-6.1.0/lib/x64Linux3gcc5.4.0/libnddscd.so(DDS_DataWriter_write_w_timestamp_untyped_generalI+0x9a0) [0x7f7530ab1a35]
+    #08    /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0) [0x7f752f6ba840]
+    #09    ./logging_publisher() [0x403039]
+```
+
+- **Timestamp**: The time when the message was logged.
+
+```plaintext
+[2020-11-27 13:51:52.503762]
+```
+
+-   **Thread Name**: Thread identification strings uniquely identify active threads
+when a message is output to the console. A thread may be a user (application)
+thread or one of several types of internal threads.
+
+```plaintext
+U00007f753124c700_logging_publish
+```
+
+-   **Activity Context**: Activity Context provides more context about a logging
+message. It is a group of resources and activities associated with an action.
+In the following example, the context is "a DataWriter is writing a sample".
+    -   Resource: the DataWriter writing the sample. The attributes of the DataWriter
+    will be guid, name, kind, topic, type and the domain id.
+    -   Activity will be “write a sample”.
+
+```plaintext
+[0x01016FA9,0xB965C9C1,0x4E59B461:0x80000003{E=DW,T=Example logging,C=logging,D=0}|WRITE]
+```
+
+- **Module**: Id representing the module
+
+```plaintext
+Mx16
+```
+
+- **File and Line**
+
+```plaintext
+writer_history.1.0/srcC/memory/Memory.c:7411
+```
+
+- **Method** name where the message is logged
+
+```plaintext
+WriterHistoryMemoryPlugin_addSample:RTI0x2161002:out of order
+```
+
+- **Message Id**: It is a identification of the message.
+
+```plaintext
+RTI0x2161002
+```
```

Comment:
Is there a section we can link to in the UM or API reference? And a setence like you have in the other examples explaining that you can get more information in the API reference and User's Manual document